magnet_coils.py

```diff
@@ -19,12 +23,9 @@ def mesh_magnets(vol_ids, logger):
 
     logger.info('Exporting coil mesh...')
     
-    # Assign export paths
-    cwd = os.getcwd()
-    base_name = 'coil_mesh'
-    general_export_path = f"{cwd}/{base_name}"
-    exo_path = f'{general_export_path}.exo'
-    h5m_path = f'{general_export_path}.h5m'
+    # Define export paths
+    exo_path = str(Path(export_dir) / 'coil_mesh.exo')
+    h5m_path = str(Path(export_dir) / 'coil_mesh.h5m')
```

Comment:
Do you need to convert to `str`? Have you tried without? If you change all path handling to use `pathlib` maybe this works fine?  Maybe paths are promoted to strings when necessary?  Just curious...

Either way, it could be easier to do eveything with pathlib until the last minute when you don't

Reply:
I noticed in L#541 below that you don't convert to string first...

Reply:
Cubit can accept `Path` objects but for some reason MOAB and CadQuery only accept strings

Reply:
Can you promote them to string at the last minute - as you pass them into the MOAB/CQ functions?  that will let you preserve them as Path objects everywhere else

Reply:
Certainly!